Resources/config/doctrine/ScheduledCommand.orm.yml

```diff
@@ -62,6 +62,12 @@ JMose\CommandSchedulerBundle\Entity\ScheduledCommand:
             unique: false
             nullable: false
             column: PRIORITY
+        executionMode:
+            type: string
+            length: 50
+            unique: false
+            nullable: false
```

Comment:
As you are aware from PR 71, this is not backward compatible.  It could be made a non-BC with some dirty logic, otherwise it should target a 1.3 branch (which one doesn't exist)